hermes/crates/cbork/.config/nextest.toml

```diff
@@ -0,0 +1,49 @@
+# cspell: words scrollability testcase
```

Comment:
Can we move cspell words here and in the other files to [project.dic](https://github.com/input-output-hk/hermes/blob/main/.config/dictionaries/project.dic)?

Reply:
`testcase` is general enough it should be in `project.dic`.
We should only really use local word lists for words that are unusual and self contained in the file.
Author surnames is a common example I come across.
`scrollability` can just be changed in the source to `scroll ability` even if it came from some other source.
Check the other spellings for these kinds of things.

Reply:
common spelling moved to project words